web/src/main/webapp/my-app/layout/partials/app-header-options.html

```diff
@@ -22,7 +22,7 @@
   <div ng-if="renderMe">
     <!-- MEDIUM AND LARGE SCREENS -->
     <div layout="row" layout-align="end center" hide-xs>
-      <md-button ng-href="apps" class="md-primary">
+      <md-button ng-href="apps" class="md-primary" aria-label="Add more apps to home">
```

Comment:
How about `aria-label="Browse app directory"` so that the same `aria-label` makes sense whether in guest mode or not?